app/models/rent.rb

```diff
@@ -0,0 +1,9 @@
+class Rent < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+
+  validates :user, presence: true
+  validates :book, presence: true
+  validates :start_at, presence: true
+  validates :end_at, presence: true
```

Comment:
good good...but we are missing a couple validations here..what if an user tries to loan a book with a start_date in the past? or a return date lower than the start date?